Multisig/UI/App/RemoteNotificationHandler.swift

```diff
@@ -165,6 +178,11 @@ class RemoteNotificationHandler {
         DispatchQueue.main.async {
             log("Saving authorization status")
             self.authorizationStatus = status
+
+            // At the time when permission granted the token will be already set so we need to register all stored safes
+            if self.authorizationStatus != nil {
```

Comment:
I  think this is not needed here, because the registerAll will be called upon token refresh anyway

Reply:
The thing is I tried it, when permission granted the token won't be refreshed,

Reply:
having this in  the "setStatus" is the wrong place  to have it. It is better  to move it up above the line 172 -  when the authorization granted, then trigger registering  all.

Anyway, let's merge and I'll test and fix it